adapters.go

```diff
@@ -134,42 +150,44 @@ func (a *whisperAdapter) handleRetrievedMessages(messages []*whisper.ReceivedMes
 
 		hlogger := logger.With(zap.Binary("hash", shhMessage.Hash))
 
-		hlogger.Debug("handling a received message")
+		hlogger.Debug("received a public message")
```

Comment:
How do we know it's a public message? `handleRetrievedMessages` handles only public messages?

Reply:
sorry, wrong log, I botched the rebase